CMakeLists.txt

```diff
@@ -72,11 +72,25 @@ else()
     add_subdirectory(.cppan)
 endif()
 
-hunter_add_package(OpenCL)
-find_package(OpenCL CONFIG REQUIRED)
-
 option(BUILD_TRAINING_TOOLS "Build training tools" ON)
 option(Tesseract_BUILD_EXE "Build tesseract executable" OFF)
+option(Tesseract_USE_OPENCL "Build tesseract with OpenCL support" ON)
+
+if(Tesseract_USE_OPENCL)
```

Comment:
You can avoid having two similar `if(Tesseract_USE_OPENCL)` condition blocks by moving this code after target creation.